src/link/link-menu.ts

```diff
@@ -51,7 +51,9 @@ export function LinkMenu(editor: MinidocToolbarEditor) {
     if (a) {
       Dom.assignAttrs({ href }, a);
     } else {
-      range?.insertNode(h('a', { href }, range.collapsed ? href : range.extractContents()));
+      range?.insertNode(
```

Comment:
We don't really. There are "cards" which hook into components and re-write them. Maybe we can use those? I'm not sure if they support inline elements, though, so that might be more work than it's worth.

An alternative is to not do it at all via minidoc internals, but instead just wait for the doc to load (e.g. in the lesson view) and then iterate over the links in the content and set the target: `contentEl.querySelectorAll('a').forEach((a) => { a.target = '_blank'; });`

Reply:
I think the best option is having `defaultLinkTarget` parameter, and setting the `target` an insert-time concern. Iterating the document and modifying links on each render sounds like overkill to me for this simple thing.

Reply:
Not disagreeing, but we do already iterate the doc at render time to mount all the cards, so it wouldn't be a big change to also add a possible transformation hook there.

But that said, I'm fine with the `defaultLinkTarget` thing. We should add `rel="noopener noreferrer"`, too, to avoid a security issue.

Probably, the link plugin should extend the minidoc signature to have the `defaultLinkTarget` property, rather than adding into the core.